tests/cases/eth2eth/scripts/init-rly

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -eu
+
+SCRIPT_DIR=$(cd $(dirname ${BASH_SOURCE:-$0}); pwd)
+source ${SCRIPT_DIR}/util/relayer-util
+
+rm -rf ${FIXTURES_DIR}
```

Comment:
It seems that this test case doesn't use fixture.

Reply:
Remove FIXTURES_DIR variable
https://github.com/datachainlab/yui-relayer-build/commit/9ad5aca503830fc1d084dc839397f39de10ad8bb